interactions/client.py

```diff
@@ -130,13 +134,11 @@ def decorator(coro: Coroutine) -> Any:
                 # so that it gives back what is/isn't.
 
                 _payload = {
-                    "type": type,
+                    "type": _type if not isinstance(_type, ApplicationCommandType) else _type.value,
```

Comment:
Please change to `_type.value if isinstance(_type, ApplicationCommandType) else _type`. This isn't incorrect code, it's just that my anxiety goes through the roof reading an inversion and it bothers me.

Reply:
Fixed as of 725b9cd.